.editorconfig

```diff
@@ -15,7 +15,7 @@ charset = utf-8
 # 4 space indentation
 [*.py]
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
🤢

Reply:
Since I switched to `black` this isn't used anymore. And `black` enforces 4 spaces anyway 🤷‍♀️